app/src/main/java/zapsolutions/zap/fragments/SettingsFragment.java

```diff
@@ -144,13 +145,14 @@ public boolean onPreferenceClick(Preference preference) {
             }
         });
 
-
+        /* Comment this out when multi wallet support is activated.*/
         // Action when clicked on "reset connection settings"
         final Preference prefResetConfig = findPreference("resetConfig");
         prefResetConfig.setOnPreferenceClickListener(new Preference.OnPreferenceClickListener() {
             @Override
             public boolean onPreferenceClick(Preference preference) {
                 if (PrefsUtil.isWalletSetup()) {
+                    WalletConfigsManager.getInstance().setConnectionToModify(PrefsUtil.getCurrentWalletConfig());
```

Comment:
Was this just for testing?
I guess as stated in the comment, with multi-wallet the option should be removed?